exasol_integration_test_docker_environment/lib/test_environment/database_waiters/is_database_ready_thread.py

```diff
@@ -44,14 +46,16 @@ def run(self):
     def create_db_connection_command(self):
         username = self.database_credentials.db_user
         password = self.database_credentials.db_password
-        connection_options = f"""-c '{self._database_info.host}:{self._database_info.db_port}' -u '{username}' -p '{password}'"""
-        cmd = f"""$EXAPLUS {connection_options}  -sql 'select 1;' -jdbcparam 'validateservercertificate=0'"""
+        connection_options = f"""-c 'localhost:{self._database_info.db_port}' -u '{username}' -p '{password}'"""
+
+        exaplus = f"/usr/opt/EXASuite-7/EASolution-{self.docker_db_image_version}/bin/Console/exaplus"
```

Comment:
Not sure if this is safe to find `exaplus`. Other option would be to call `find`.